cord19-explorer/src/main/webapp/WEB-INF/views/paper.html

```diff
@@ -64,9 +64,13 @@ <h3 th:text="*{section.title != ''} ? *{section.title}: 'No Section title'"></h3
 	      <span th:each="para : ${section.paragraphs}">
 	      <script th:inline="javascript" th:if="${not #lists.isEmpty(para.locationMatches)}">
 	      $(function() {
-	      	[# th:each="match : ${para.locationMatches}" ]
+	      	[# th:each="match, iStat : ${para.locationMatches}" ]
 	      	[# th:if="${match != null && match.selectedArticle != null}" ]
-	      	$("#[(${#ids.next('para')})]").mark("[(${match.locationName})]", { "separateWordSearch": false})	
+	      	$("#[(${#ids.next('para')})]").mark("[(${match.locationName})]", { "separateWordSearch": false})
+	     	var title = [[${match.selectedArticle.title}]]
+	      	$("#[(${#ids.next('para')})]").mark("[(${match.locationName})]", { "separateWordSearch": false}).attr('title', title);
+	        $("#[(${#ids.next('para')})]").mark("[(${match.locationName})]", { "separateWordSearch": false, "className": "wiki-[(${#ids.next('para')})]-[(${iStat.index})]"})
```

Comment:
you are marking the text 3 times resulting in 3 divs around the text. One should be enough